user-console/src/main/java/org/pentaho/mantle/client/MantleApplication.java

```diff
@@ -484,6 +486,12 @@ public DeckPanel getContentDeck() {
     return contentDeck;
   }
 
+  public static UserDropDown getUserDropDown(){
```

Comment:
javadoc comment for MatleApplication#getUserDropDown(). mention is implanted to do lazy loading.

Does this function MatleApplication#getUserDropDown() access level need to be "public" I don't see any external class calling this function?